doc/api/deprecations.md

```diff
@@ -660,6 +660,15 @@ Type: Runtime
 
 `REPLServer.parseREPLKeyword()` was removed from userland visibility.
 
+<a id="DEP00XX"></a>
+### DEP00XX: path._makeLong()
+
+Type: Runtime
```

Comment:
-1 on runtime-deprecating without a reason, **especially** if we’re exposing an alias